selectolax/lexbor/node.pxi

```diff
@@ -1039,23 +1042,77 @@ cdef class LexborNode:
         Returns
         -------
         bool
-            ``True`` when the node is a text node and
-            ``lxb_dom_node_is_empty`` reports that its parent subtree contains
-            only whitespace (or nothing).
+            ``True`` when the node is a text node whose character data consists
+            only of ASCII whitespace characters (space, tab, newline, form feed
+            or carriage return).
         """
         return self._is_empty_text_node(self.node)
 
-    cdef inline bint _is_empty_text_node(self, lxb_dom_node_t *node):
-        if node.type != LXB_DOM_NODE_TYPE_TEXT:
+    cdef inline bint _is_empty_text_node(self, lxb_dom_node_t *text_node):
```

Comment:
There is no need to keep it as a method. Passing self is not free, and you don't use it. Please refactor it into a function and put it in the utils module. Same for `_is_whitespace_only`